--- origsrc/xfce4-verve-plugin-1.0.0/panel-plugin/Makefile.am	2010-12-20 10:13:00.000000000 -0600
+++ src/xfce4-verve-plugin-1.0.0/panel-plugin/Makefile.am	2011-03-13 00:42:52.202897200 -0600
@@ -17,18 +17,18 @@ xfce4_verve_plugin_CFLAGS = 						\
 	-I$(top_srcdir)							\
 	-DPACKAGE_LOCALE_DIR=\"$(localedir)\"				\
 	@LIBEXO_CFLAGS@							\
-	@LIBXFCE4PANEL_CFLAGS@						\
 	@LIBXFCE4UTIL_CFLAGS@						\
 	@LIBXFCEGUI4_CFLAGS@						\
+	@LIBXFCE4PANEL_CFLAGS@						\
 	@LIBPCRE_CFLAGS@						\
 	@GLIB_CFLAGS@							\
 	@GTHREAD_CFLAGS@						
 
 xfce4_verve_plugin_LDADD = 						\
 	@LIBEXO_LIBS@							\
-	@LIBXFCE4PANEL_LIBS@						\
 	@LIBXFCE4UTIL_LIBS@						\
 	@LIBXFCEGUI4_LIBS@						\
+	@LIBXFCE4PANEL_LIBS@						\
 	@LIBPCRE_LIBS@							\
 	@GLIB_LIBS@							\
 	@GTHREAD_LIBS@
